### ISSUE TITLE
Plugin doesn't work normally after grafana upgrading to version 9 

### ISSUE THREAD

**Issue by PolinaVens** [camptocamp/grafana-prometheus-alertmanager-datasource#143]
**Grafana version:** 9.2.3

**Plugin version:** 1.2.1

**Error:**
Popup says:
Error updating options: Couldn't find a query runner that matches supplied arguments.

**JS Console log:**
```
Error: Couldn't find a query runner that matches supplied arguments.
    getRunnerForDatasource queryRunners.ts:62
    onNewRequest VariableQueryRunner.ts:115
    RxJS 6
    queueRequest VariableQueryRunner.ts:76
    jm actions.ts:50
    jm actions.ts:44
actions.ts:893:16
```

**Comment by jtheory** [comment]
If it helps -- the core Grafana code base does include a datasource plugin for a Prometheus-compatible Alertmanager; it's built-in now.
Just search for "Alertmanager" in data sources.

code: https://github.com/grafana/grafana/tree/main/public/app/plugins/datasource/alertmanager

See if that works for you. :crossed_fingers:

**Comment by rsauron** [comment]
The newer grafana 9.x branch does have a built in datasource for Alertmanager. but it does not let you query it or build dashboards from it. The datasource can be used for a "Alert list" panel is the only use case i can find. Maybe someone knows better?

**Comment by antoinedeschenes** [comment]
Indeed, the Grafana 9.x datasource doesn't allow usage from dashboards. Alert list might work if the Alerts section is enabled, but we have no interest in managing our alerts through Grafana, so we keep it disabled.

**Comment by hbollon** [comment]
Hello, can you give a try to the new v2.0.0 release and see if you're always experiencing this issue please?
